src/lp-form.js

```diff
@@ -93,7 +96,7 @@ function lpForm (options={}) {
         initialValues: filterInitialValues(initialValues),
         onSubmit: debounceSubmit ? debounce(wrappedOnSubmit, debounceSubmit) : wrappedOnSubmit,
         onChange: submitOnChange ? createSubmittingOnChange(onChange) : onChange,
-        validate: validate(constraints),
+        validate: validateOptions ? values => validateWithOptions(constraints, values, validateOptions) : validate(constraints),
```

Comment:
We can keep the `validateOptions={}` and always use `validateWithOptions` here:
`validate: values => validateWithOptions(constraints, values, validateOptions)`